app/models/licence_facade.rb

```diff
@@ -38,10 +41,16 @@ def title
   end
 
   def url
-    published? ? @artefact['web_url'] : nil
+    published? ? web_url : nil
   end
 
   def short_description
-    published? ? @artefact['details']['licence_short_description'] : nil
+    published? ? @artefact['licence_short_description'] : nil
```

Comment:
If we're retiring use of Content API, should we also remove all references to `@artefact`?